docroot/themes/humsci/humsci_basic/src/js/tables/wrap.js

```diff
@@ -0,0 +1,8 @@
+// TODO: Can we just use the global jQuery?
+// $('table').wrap('<div class="hb-table-wrap"></div>');
```

Comment:
hm yeah. I kind of like using vanilla jS, but is jQuery nice since it's shorter?

Reply:
Talked to @asimpson, he agreed with using vanilla JS for any of our code. We can possibly drop jQuery at some point.